batchlib/workflows/cell_analysis.py

```diff
@@ -196,6 +197,16 @@ def run_cell_analysis(config):
                 'identifier': identifier},
             'run': {'force_recompute': False}}))
 
+        job_list.append((DbResultWriter, {
+            'build': {
+                "username": config.db_username,
+                "password": config.db_password,
+                "host": config.db_host,
+                "port": config.db_port,
+                "db_name": config.db_name
+            },
+            'run': {'force_recompute': True}}))
```

Comment:
@constantinpape if I don't pass `force_recompute: True` the job is skipped saying `no inputs left to process`; lemme know if we can leave it like this or should be use different config so that `BatchJob.get_inputs()` correctly returns hdf5 table.

Reply:
See comment above.

Reply:
:white_check_mark: